src/lib/TermBuilder.ml

```diff
@@ -271,6 +287,6 @@ struct
     lam @@ fun p ->
     cof_split
       (el fam_i)
-      d_i                  (fun q -> ap bdry [i; q])
```

Comment:
I was about to complain about this when you said you hate spaces for alignment. LOL

Reply:
😎